api/src/test/java/org/openmrs/OpenmrsTestsTest.java

```diff
@@ -111,18 +109,16 @@ public void shouldHaveClassNameEndWithTestIfContainsMethodTestAnnotations() {
 		for (Class<?> currentClass : getClasses("^.*(?<!Test|IT|PT)\\.class$")) {
 			
 			// skip over classes that are @Ignore'd
-			if (currentClass.getAnnotation(Ignore.class) == null) {
+			if (currentClass.getAnnotation(Ignore.class) == null || currentClass.getAnnotation(Disabled.class) == null) {
 				boolean foundATestMethod = false;
 				
 				for (Method method : currentClass.getMethods()) {
-					if (method.getAnnotation(Test.class) != null) {
+					if (method.getAnnotation(org.junit.Test.class) != null && method.getAnnotation(Test.class) != null) {
```

Comment:
Shouldnt this also be an `||`?
Since before it was setting `foundATestMethod` to true if there is a method with the `org.junit.Test` annotation. I think now it should also count the ones with the `org.junit.jupiter.api.Test` annotation as a test method. Both annotations will never be present so an `&&` cannot identify a test method then.

therefore

```suggestion
					if (method.getAnnotation(org.junit.Test.class) != null || method.getAnnotation(Test.class) != null) {
```

Reply:
![Screenshot from 2020-07-09 19-21-29](https://user-images.githubusercontent.com/39851246/87076337-8506ae80-c219-11ea-9417-77cff4f77a8f.png)
@teleivo  here is the error I got.